paket.dependencies

```diff
@@ -1,35 +1,59 @@
 source http://nuget.org/api/v2
+redirects: on
+strategy: max
 
-nuget FSharp.Compiler.Service
 nuget Fantomas
+nuget FSharp.Compiler.Service
+nuget FSharp.Core
+nuget FSharp.Management
 nuget FSharp.ViewModule.Core
-nuget FsXaml.Wpf
-nuget SourceLink.SymbolStore 0.5.0
 nuget FSharpLint.Core prerelease
-nuget VSSDK.IDE.Deploy.14
-nuget FSharp.Management
 nuget FsPickler
-nuget FSharp.Core redirects:on
+nuget FsXaml.Wpf
+nuget SourceLink.SymbolStore 0.5.0
 
+group vssdklegacy
+    source http://nuget.org/api/v2
+    framework >= net40
+    
+    nuget Microsoft.VisualStudio.Shell.Interop.8.0
+    nuget Microsoft.VisualStudio.Shell.Immutable.10.0
+    nuget VSSDK.ComponentModelHost.11
+    nuget VSSDK.DTE
+    nuget VSSDK.DTE.8
+    nuget VSSDK.DTE.10
+    nuget VSSDK.NavigateTo
+    nuget VSSDK.VSLangProj
+    
+    
+group vssdk
+    source http://nuget.org/api/v2
+    framework >= net45
+    
+    nuget Microsoft.VisualStudio.Editor
+    nuget Microsoft.VisualStudio.Language.Intellisense
```

Comment:
> Is it related to GAC? Is there any way to check that value?

I don't believe so.  

> the truth is in paket.lock and this file isn't touched unless someone invokes the install or upgrade commands, we can also specify < 15 in paket.dependencies as well which will in turn avoid install or upgrade to look at eventual versions 15 or above.

I wish there was a stronger way to prevent this.  But if not then we just need to be very diligent about the contents of the lock file here.